stork/src/base/StorkApp.C.module

```diff
@@ -0,0 +1,47 @@
+#include "StorkApp.h"
```

Comment:
This file was left over by the script. It should be deleted when I create an app.